bindata/v4.1.0/kube-apiserver/check-endpoints-role.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: check-endpoints
```

Comment:
this part can be a clusterrole so you can re-use it.

Also, the name `system:openshift:operator?:check-endpoints`